pom.xml

```diff
@@ -58,12 +58,12 @@ under the License.
     <url>https://github.com/apache/maven-archetypes/tree/${project.scm.tag}</url>
   </scm>
   <issueManagement>
-    <system>jira</system>
-    <url>https://issues.apache.org/jira/browse/MARCHETYPES</url>
+    <system>GitHub</system>
+    <url>https://github.com/apache/maven-archetypes/issues</url>
   </issueManagement>
   <ciManagement>
     <system>Jenkins</system>
-    <url>https://ci-builds.apache.org/job/Maven/job/maven-box/job/maven-archetypes/</url>
+    <url>https://ci-maven.apache.org/job/Maven/job/maven-box/job/maven-archetypes/</url>
```

Comment:
Oh I never change that, do I have to update it too, when I pick up those files?

Reply:
only in this case ... I did it by the way
in most of project link to Jenkins is ok